config.tmp.yml

```diff
@@ -31,3 +31,5 @@ telegram_bot:
   debug: false
   offset: 0
   timeout: 60
+  white_lists: ["-1002437731689"] # list id chat in tele will receive daily random image
```

Comment:
Keep in mind that we don't use real value here!

Reply:
Agree